phonenumber_field/modelfields.py

```diff
@@ -85,6 +89,11 @@ def formfield(self, **kwargs):
         }
         defaults.update(kwargs)
         return super(PhoneNumberField, self).formfield(**defaults)
+        
+    def deconstruct(self):
```

Comment:
A `deconstruct` method is required from django 1.7 for all custom fields but PhoneNumberField is based on very common `CharField`. Moreover this particular `deconstruct` method does nothing... well it removes `max_length` parameter which is wrong. PhoneNumberField uses 128 by default but if you want you can set it to different value. Deleting `max_length` parameter you are loosing track of what user's defined.